### DISCUSSION TITLE
Windows Development

### DISCUSSION THREAD

**Discussion by kaaax0815** [wg-easy/wg-easy#1318] [Q&A] [answered]
I am using Windows 11, Docker Desktop and WSL 2 Ubuntu to rewrite the UI in nuxt (`feat-ui`).
On a Macbook I have no problems. But with Windows I get weird errors, where the server is not starting properly or where there are generally weird issues.
This this also happen to anyone else? Or is Docker on Windows just shit?

**Comment by kaaax0815** [comment]
I am thinking about dual booting Linux. Any recommendations for Distros? Or has anybody feedback on this approach

**Comment by afunworm** [comment, marked as answer]
I don't think I can provide any details to help you solve the problem, but I generally experience a lot of inconsistency and stability issues with Docker Desktop, whether it's on Windows or Linux. Uninstalling Docker Desktop and just using Docker generally solves a lot of the issues.
